aws-iam-role-policies/module_test.go

```diff
@@ -0,0 +1,31 @@
+package test
+
+import (
+	"testing"
+
+	"github.com/chanzuckerberg/go-misc/tftest"
+)
+
+func TestAWSIAMROle(t *testing.T) {
+	roleName := tftest.CreateRole(t)
+	project := tftest.UniqueID()
+	env := tftest.UniqueID()
+	service := tftest.UniqueID()
+	owner := tftest.UniqueID()
+	defer tftest.DeleteRole(t, roleName) //nolint
+
+	terraformOptions := tftest.Options(
+		tftest.IAMRegion,
+		map[string]interface{}{
+			"role_name": roleName,
+			"project": project,
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
File is not `gofmt`-ed with `-s` (gofmt)